addons/dragging/functions/fnc_handleScrollWheel.sqf

```diff
@@ -37,7 +37,9 @@ _position set [2, ((_position select 2) + _scrollAmount min (_maxHeight + 1.5))
 
 // move up/down object and reattach at current position
 detach _carriedItem;
-private _positionChange =  _position vectorDiff (getPosATL _carriedItem);
+
+// Uses this method of selecting position because setATL dosen't work on dedicated server
```

Comment:
`doesn't`

That's not entirely true though, it only doesn't work correctly when object is not local, if I understood the core issue correctly.